crates/miden-lib/src/transaction/inputs.rs

```diff
@@ -148,6 +149,9 @@ impl TransactionAdviceInputs {
         self.extend_stack([Felt::from(tx_inputs.input_notes().num_notes())]);
         self.extend_stack(tx_args.tx_script().map_or(Word::empty(), |script| script.root()));
         self.extend_stack(tx_args.tx_script_arg());
+
+        // --- auth procedure args key -------------------------------------------
+        self.extend_stack(tx_args.auth_arg());
```

Comment:
Related to the above comment. I would rename `tx_script_arg()` to `tx_script_args()`.